src/Cart.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { connect } from 'react-redux';
+import Modal from './UI/Modal';
+import './Cart.css';
+import { BsFillTrashFill } from "react-icons/bs";
+
+const Cart = (props) => {
+
+    const [ordering, setOrdering] = useState(false);
+    const [text, setText] = useState();
+
+    const orderCart = () => {
+        let order = props.cart.map(product => {
+            return {
+                id: product.id,
+                amount: product.amount
+            }
+        });
+        setOrdering(true)
+        axios.post('http://localhost:8080/order', {products: order})
+        .then(response => {
+            const data = response.data;
+            setText('Successfully placed the order id:' + data.id + ' total price: ' + data.totalPrice);
```

Comment:
Instead of connect many strings, you can use `` quotation marks and enter the variables values in ${}

(like: `total price: ${data.totalPrice}`)